Assets/library/js/builder.service.js

```diff
@@ -221,6 +221,9 @@ export default class BuilderService {
         'gjs-plugin-ckeditor': BuilderService.getCkeConf(),
       },
     });
+    
+    // make editor accessable globaly for customizing
```

Comment:
```suggestion
    // make editor accessible globally for customizing
``` 

I know it's only a comment, but a couple of typos to fix ;)